packages/create-vite/template-react/eslint.config.js

```diff
@@ -17,18 +16,14 @@ export default [
         sourceType: 'module',
       },
     },
-    settings: { react: { version: '18.3' } },
     plugins: {
-      react,
       'react-hooks': reactHooks,
       'react-refresh': reactRefresh,
     },
     rules: {
       ...js.configs.recommended.rules,
-      ...react.configs.recommended.rules,
-      ...react.configs['jsx-runtime'].rules,
       ...reactHooks.configs.recommended.rules,
-      'react/jsx-no-target-blank': 'off',
+      'no-unused-vars': ['error', { varsIgnorePattern: '^[A-Z]' }],
```

Comment:
Should we also add _ which match common practice?
```suggestion
      'no-unused-vars': ['error', { varsIgnorePattern: '^[A-Z_]' }],
```

Reply:
Good call. Yeah that makes sense